### PR TITLE
584 after duplicating a token take user straight to the token edit form

### PR DESCRIPTION
**Pull request by swordEdge**
- [x] Functional review: @six7

### CODE REVIEW

**Comment by six7** [comment]
@swordEdge this isn't working yet. The EditTokenForm pops up, which is as expected, but hitting `Duplicate` doesn't do anything.

**Comment by six7** [comment]
Functionality approved!

File: src/app/components/TokenTree.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import AddIcon from '@/icons/add.svg';
 
 export type ShowFormOptions = {
   name: string;
-  isPristine?: boolean;
+  status?: string;
```
Comment (LiamMartens) [inline src/app/components/TokenTree.tsx:13]: Should this be typed as `EditTokenFormStatus` perhaps?

File: src/types/tokens/EditTokenObject.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ import type { TokenTypeSchema } from './TokenTypeSchema';
 
 type EditTokenObjectProperties = {
   initialName: string;
-  isPristine: boolean;
+  status: string;
```
Comment (LiamMartens) [inline src/types/tokens/EditTokenObject.ts:24]: Same here, this should probably be the enum you created.

**Review by LiamMartens — CHANGES_REQUESTED** [review]

**Review by six7 — APPROVED** [review]

**Pull request merged**